opengever/maintenance/scripts/list_dossiers_shared_with_group.py

```diff
@@ -72,18 +80,18 @@ def main():
 
     groupname = args[0]
     group = api.group.get(groupname)
-    if not group:
+    if not group and not options.force:
         print "Group does not exist"
         print "Available groups"
         print map(lambda group: group.id, api.group.get_groups())
         sys.exit(1)
 
-    dossier_lister = DossierSharedWithGroupLister(group)
+    dossier_lister = DossierSharedWithGroupLister(groupname)
     dossier_lister.list_dossiers_shared_with_group()
     dossier_lister.print_table()
 
     log_filename = LogFilePathFinder().get_logfile_path(
-        'list_dossiers_shared_with_{}'.format(group), extension="csv")
+        'list_dossiers_shared_with_{}'.format(groupname), extension="csv")
```

Comment:
we could consistently name the variable accross the script. so use `group_id` instead of `groupname` here, as well, as i think it is the better name.